src/invidious/user/exports.cr

```diff
@@ -0,0 +1,35 @@
+struct Invidious::User
+  module Export
+    extend self
+
+    def to_invidious(user : User)
+      playlists = Invidious::Database::Playlists.select_like_iv(user.email)
+
+      return JSON.build do |json|
+        json.object do
+          json.field "subscriptions", user.subscriptions
+          json.field "watch_history", user.watched
+          json.field "preferences", user.preferences
+          json.field "playlists" do
+            json.array do
+              playlists.each do |playlist|
+                json.object do
+                  json.field "title", playlist.title
+                  json.field "description", html_to_content(playlist.description_html)
+                  json.field "privacy", playlist.privacy.to_s
+                  json.field "videos" do
+                    json.array do
+                      Invidious::Database::PlaylistVideos.select_ids(playlist.id, playlist.index, limit: 500).each do |video_id|
```

Comment:
Hmm, that 500 video limit here is a potential problem (instance admins can configure playlists to have more than 500 videos now). I'm aware that this was present in the original code, so can you put a higher number for now?

Reply:
I didn't know about the config, I will use that value instead

Reply:
Oh, right, that's even better ^^